src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -125,45 +125,49 @@ export default defineConfig({
 })
 ```
 
-### platformProxy
+### `platformProxy`
 
 Determines whether and how the Cloudflare runtime is added to `astro dev`. It contains proxies to local `workerd` bindings and emulations of Cloudflare specific values, allowing the emulation of the runtime in the Node.js dev process. Read more about the [Cloudflare Runtime](#cloudflare-runtime).
 
 :::note
 Proxies provided by this are a best effort emulation of the real production. Although they are designed to be as close as possible to the real thing, there might be a slight differences and inconsistencies between the two.
 :::
 
-#### platformProxy.enabled
+#### `platformProxy.enabled`
 <p>
-**Type:** `{ enabled?: boolean }`<br />
-**Default:** `{ enabled: false }`
+**Type:** `boolean`<br />
+**Default:** `true`
 </p>
 
-The `enabled` property allows you to enable the Cloudflare runtime in `astro dev`.
+Determines whether to enable the Cloudflare runtime in development mode.
 
-#### platformProxy.configPath
+#### `platformProxy.configPath`
 <p>
-**Type:** `{ configPath?: string }`<br />
-**Default:** `{ configPath: 'wrangler.toml' }`
+**Type:** `string`<br />
+**Default:** `undefined`
 </p>
 
-The `configPath` property allows you to define your Wrangler configuration file, relative to the root of your Astro project. 
+Defines the path to the Wrangler configuration file. If no value is set, it tracks `wrangler.toml`, `wrangler.json`, and `wrangler.jsonc` in the project root.
 
-#### platformProxy.environment
+#### `platformProxy.environment`
 <p>
-**Type:** `{ environment?: string }`<br />
-**Default:** `{ environment: undefined }`
+**Type:** `string`<br />
+**Default:** `undefined`
 </p>
 
-The `environment` property allows you to define which [environment configuration](https://developers.cloudflare.com/workers/wrangler/environments/) in your Wrangler configuration to use.
+Sets the [Cloudflare environment](https://developers.cloudflare.com/workers/wrangler/environments/) to use. You must select the environment defined in the Wrangler configuration file, otherwise an error occurs.
```

Comment:
small nit, from what I understood you can have several environments set in a Wrangler configuration, so saying "the environment defined" sounds a bit misleading, as if you can only have one environment to select from. 
```suggestion
Sets the [Cloudflare environment](https://developers.cloudflare.com/workers/wrangler/environments/) to use. You must select an environment defined in the Wrangler configuration file, otherwise an error occurs.
```

Reply:
Thank you!